### PR TITLE
Add Stanford POS Tagger to NLPStack.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
I moved the Stanford POS models into the datastore so I could remove all the Stanford models from the build. @mhrmm fyi.

**Comment by mhrmm** [comment]
Thanks, Michael. FYI @dirkgr, no need to release a new version of nlpstack with these changes until I train a new parser model that uses these tags.

File: tools/postag/src/main/scala/org/allenai/nlpstack/postag/StanfordPostagger.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+object StanfordPostagger {
+  def loadDefaultModel(): MaxentTagger = {
+    val filePath = Datastore.directoryPath(
+      "edu.stanford.nlp.models.pos-tagger",
+      "english-left3words-3.4.1",
+      1
+    )
+    new MaxentTagger(filePath.toString + "/english-left3words-distsim.tagger")
```
Comment (mhrmm) [inline tools/postag/src/main/scala/org/allenai/nlpstack/postag/StanfordPostagger.scala]: Should be: 

new MaxentTagger(filePath.toString + "/english-left3words/english-left3words-distsim.tagger")

Comment (schmmd) [inline tools/postag/src/main/scala/org/allenai/nlpstack/postag/StanfordPostagger.scala]: Um... Isn't that what's there?

Comment (mhrmm) [inline tools/postag/src/main/scala/org/allenai/nlpstack/postag/StanfordPostagger.scala]: Not quite.

**Comment by mhrmm** [comment]
Closing in favor of PR #113, which has these changes merged in.

**Pull request closed without merging**